### DISCUSSION TITLE
Docker / Docker Compose on QNAP - HomeKit can't be added

### DISCUSSION THREAD

**Discussion by idominiki** [koush/scrypted#1382] [Q&A] [answered]
Hi together!

I have setup Scrypted via Docker or Docker Compose in my QNAP NAS. I have the problem, that cameras or even the bridge can't be added to Home App. 

After I scan the QR code the device is gone. 

I set the server address for Scrypted to my NAS address, as Container is running in host mode. 

Has anyone the same problem?

![image](https://github.com/koush/scrypted/assets/62521486/1173b03b-2093-4767-8a0b-cc38a2ea598e)
![image](https://github.com/koush/scrypted/assets/62521486/ea2199cc-cc5e-4f5f-8399-8b9e45a9fa8a)
![image](https://github.com/koush/scrypted/assets/62521486/dcca2b36-3d56-41b5-bd62-82ce3327cd31)
![image](https://github.com/koush/scrypted/assets/62521486/b7feb341-8072-4d45-8502-9584577f3034)
![image](https://github.com/koush/scrypted/assets/62521486/8297e2af-67eb-4723-9131-8992a1170a5d)

**Comment by lydigaga** [comment]
I am encountering the same problem, I can detect them, but I am unable to add them to the homekit

**Comment by koush** [comment, marked as answer]
not familiar with qnap. might need to use bridge network mode?

**Reply by idominiki** [reply]
@koush @lydigaga Just configured Scrypted in Bridge Mode with static IP and all works fine now!
